python/frontend/cirrus/cirrus/core.py

```diff
@@ -135,14 +150,17 @@ def get_updates_per_second(self, fetch=True):
 
         num_lambdas = self.get_num_lambdas()
         self.get_updates_per_second()
-        num_task = 3
 
-        if num_lambdas == None:
+        if num_lambdas == None: # not clear when this can happen. It seems to me self.get_num_lambdas() never returns None
```

Comment:
@yangryan0 	Can you please check this?

Reply:
self.get_num_lambdas returns none when there is no response from the parameter server or the connection times out.